client/src/app/shared/shared-main/shared-main.module.ts

```diff
@@ -81,10 +81,9 @@ import { AUTH_INTERCEPTOR_PROVIDER } from './auth'
 
     HelpComponent,
     ListOverflowComponent,
+    TopMenuDropdownComponent,
 
     UserNotificationsComponent,
-
-    FeedComponent
```

Comment:
?

Reply:
It is already present a few lines above.